app/models/cluster_group.rb

```diff
@@ -26,6 +26,9 @@ class ClusterGroup
   field :subsampled, type: Boolean, default: false
   field :is_subsampling, type: Boolean, default: false
 
+  # denotes when image_pipeline has been run for this cluster_group
+  field :has_image_data, type: Boolean, default: false
```

Comment:
Blocker: Clearer, less ambiguous intent:

```suggestion
  field :has_image_cache, type: Boolean, default: false
```